Doc/tutorial/classes.rst

```diff
@@ -475,12 +475,20 @@ Random Remarks
 
 .. These should perhaps be placed more carefully...
 
-Data attributes override method attributes with the same name; to avoid
-accidental name conflicts, which may cause hard-to-find bugs in large programs,
-it is wise to use some kind of convention that minimizes the chance of
-conflicts.  Possible conventions include capitalizing method names, prefixing
-data attribute names with a small unique string (perhaps just an underscore), or
-using verbs for methods and nouns for data attributes.
```

Comment:
Wow.  At first I was confused.  If one defines module, class, or instance attributes with the same name, the second wins.  Then the first two advice items are wrong, the third correct, but misses the realistic case of noun data attribute duplication on class and instance.  Your example covers that nicely. Please apply and backport.